railties/lib/rails/generators/testing/behavior.rb

```diff
@@ -7,6 +7,7 @@
 require "active_support/testing/stream"
 require "active_support/concern"
 require "rails/generators"
+require "rails/deprecator"
```

Comment:
Since `rails/generators/testing/behavior` requires `rails/generators`, and `rails/generators` [requires `rails/command`](https://github.com/rails/rails/blob/dcc75f99858503de51a9fab5108df4396a9de968/railties/lib/rails/generators.rb#L4), I think it would be good to require `rails/deprecator` [in `rails/command`](https://github.com/rails/rails/blob/dcc75f99858503de51a9fab5108df4396a9de968/railties/lib/rails/command.rb#L3-L6).  That way all command-related (and generator-related) files will have access to `Rails.deprecator` along with the rest of the deprecation API.  That would also match the way `railties/lib/rails.rb` [requires `active_support` + `rails/deprecator`](https://github.com/rails/rails/blob/dcc75f99858503de51a9fab5108df4396a9de968/railties/lib/rails.rb#L7-L14) for the same purpose.

Reply:
Thanks, @jonathanhefner! Updated with the suggestion.